tests/php/Command/Room/CreateTest.php

```diff
@@ -318,18 +323,6 @@ public function invalidProvider(): array {
 				],
 				"Invalid room name.\n",
 			],
-			[
-				[
-					'name' => '  ',
-				],
-				"Invalid room name.\n",
-			],
-			[
-				[
-					'name' => str_repeat('x', 256),
```

Comment:
It is now part of the RoomService and the creation throws an exception. The CreateCommand itself does not interact with the room name anymore. So it will be covered by unit tests for the RoomService later.